docs/source/conf.py

```diff
@@ -13,7 +13,7 @@
 copyright = '2022, SkyPilot Team'
 author = 'the SkyPilot authors'
 
-release = '0.1'
+release = '0.1.0'
```

Comment:
We're at 0.1.1 on pypi, should we use that here?

Reply:
Done, good catch!